core/procedures.js

```diff
@@ -108,13 +108,10 @@ Blockly.Procedures.sortProcedureMutations_ = function(mutations) {
     var procCodeA = a.getAttribute('proccode');
     var procCodeB = b.getAttribute('proccode');
 
-    if (procCodeA < procCodeB) {
-      return -1;
-    } else if (procCodeA > procCodeB) {
-      return 1;
-    } else {
-      return 0;
-    }
+    return procCodeA.localeCompare(procCodeB, undefined, {
```

Comment:
@joshyrobot, I think you can just use localeCompare with only the first argument and leave the other arguments out (here and all the other lines changed). It seems to do the right thing, and then we don't have to worry about these extra options.